### PR TITLE
Food & Fluid Plus

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
- A new *Fluid Output* field appears when `Yes - Measured` is selected for the *Passed Urine* field.
- Fluid balance is calculated and displayed on mobile table, desktop table, and reports.

### CODE REVIEW

File: nh_food_and_fluid/models/food_and_fluid.py [hunk 1]
```diff
@@ -194,23 +246,114 @@ def calculate_total_fluid_intake(self, spell_activity_id, date_time):
         :type spell_activity_id: int
         :param date_time:
         :type: str or datetime
-        :return: Total fluid intake.
+        :return: Total fluid intake or None if there are no obs for the period.
```
Comment (moderatemisbehaviour) [inline nh_food_and_fluid/models/food_and_fluid.py]: Change this docstring - it doesn't return `None` anymore.

File: nh_food_and_fluid/models/food_and_fluid.py [hunk 2]
```diff
@@ -4,19 +4,22 @@
 
 from openerp import models, api
 from openerp.addons.nh_observations import fields as obs_fields
+from openerp.addons.nh_odoo_fixes import validate
 from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT as DTF
 
 
 class NHClinicalFoodAndFluid(models.Model):
 
+    # TODO Rename to food_and_fluid? Does not exceed 63 character limit.
```
Comment (moderatemisbehaviour) [inline nh_food_and_fluid/models/food_and_fluid.py]: Decide what you're doing with this.

**Comment by moderatemisbehaviour** [comment]
LGTM :shipit:

**Comment by Gimpneek** [comment]
LMFGTM :shipit:

**Comment by Nayira** [comment]
Tests for EOBS-1121, EOBS-1127 and EOBS-1128 Passed.
Feature branch issues EOBS-1182 and EOBS-1183 confirmed as fixed.
Ship it conga! :shipit: :shipit: :shipit: :shipit: :shipit: :shipit:

**Pull request merged**